### PR TITLE
Create-Component-Use-Cases

### PR DESCRIPTION
**Pull request by CharlineLavigne**

### CODE REVIEW

File: components/UseCases.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
+import imageTop from '../assets/img/user-case-privacy-phone.png';
```
Comment (AlexikM) [inline components/UseCases.js]: The images shouldn't be imported inside of the component - but from the page displaying the component

File: components/UseCases.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    return(
+        <div className='bg-[#EAEAEA] flex flex-col md:flex-row w-[80%] m-auto'>
+            <p className='w-[60%] md:w-[40%] text-[3vw] md:text-[1.9vw] ml-10 mt-10 md:m-auto md:ml-10'>
+            {content}
+            </p>
+            <div className='w-[90%] md:w-[55%] flex p-10 m-auto'>
+                <div className='w-[48%] my-auto'>
+                    <Image 
+                    src={imageTop}
+                    alt="Use case illustration" />
```
Comment (AlexikM) [inline components/UseCases.js]: Alt cannot be the same for all pictures.

**Review by AlexikM — CHANGES_REQUESTED** [review]
Thanks a lot Kali, please check my comments

File: components/UseCases.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
+import imageTop from '../assets/img/user-case-privacy-phone.png';
+import imageBottom from '../assets/img/user-case-privacy-person.png';
+import imageSide from '../assets/img/user-case-privacy-texting.png';
+
+
+export default function UseCases(content) {
+    return(
+        <div className='bg-[#EAEAEA] flex flex-col md:flex-row w-[80%] m-auto'>
```
Comment (souksou) [inline components/UseCases.js]: Please use global variable for the styling or create a new task for theming.

File: components/UseCases.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            <p className='w-[60%] md:w-[40%] text-[3vw] md:text-[1.9vw] ml-10 mt-10 md:m-auto md:ml-10'>
```
Comment (souksou) [inline components/UseCases.js]: Please use tailwind variable for the text example: 
md:text-xl

**Review by AlexikM — APPROVED** [review]

**Pull request merged**